src/main/java/com/autotune/analyzer/experiment/ExperimentResultValidation.java

```diff
@@ -88,29 +89,28 @@ public void validate(List<ExperimentResultData> experimentResultDataList, Map<St
                                     proceed = true;
                             } else {
                                 proceed = false;
-                                resultData.setValidationResultData(new ValidationOutputData(false, errorMsg));
+                                resultData.setValidationResultData(new ValidationOutputData(false, errorMsg, HttpServletResponse.SC_BAD_REQUEST));
```

Comment:
Looks like this method call should have been `setValidationOutputData`, got missed out in #659

Reply:
Updated wherever missed